src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.card-block {
+  font-family: Roboto, sans-serif;
+  width: 166px;
+  padding: 16px;
```

Comment:
add paddings to card according to design, so then you could only margin-bottom to each element